cadc-util/src/test/java/org/opencadc/persist/EntityTest.java

```diff
@@ -85,7 +85,7 @@ public class EntityTest {
     private static final Logger log = Logger.getLogger(EntityTest.class);
 
     static {
-        Log4jInit.setLevel("org.opencadc.persist", Level.INFO);
+        Log4jInit.setLevel("org.opencadc.persist", Level.DEBUG);
```

Comment:
Might want to turn it back to `INFO`?

Reply:
DEBUG is OK here; Entity has a static boolean that actually controls the copious DEBUG logging. I will move that to the `static` block so devs can easily turn it on or off.